index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style/style.css" />
+  </head>
+  <body id="main">
+    <title id="title">Dr. Norman Borlaug</title>
+    <div id="img-div">
+      <h1>Dr. Norman Borlaugs</h1>
+      <img class="responsive" src="/img/Dr Norman.jpg" alt="Norman Borlaug" />
+      <caption id="img-caption">
+        About Dr. Norman Borlaug:
+      </caption>
+      <h5 id="tribute-info">
+        This is Dr. Norman Borlaug. He is very kind and generous human being.I
+        really appreciate his generosity and work in the area of Medical Science
+      </h5>
+    </div>
```

Comment:
This is good work but it can be better if you use semantic tags to make your code more professional and readable and make sure that your img is linked to the right folder.

Reply:
- [ ] Here it would be better if you give an example of semantic tags he/she would be using 👍 . For example, "since this is a paragraph, it needs to be in a `p` tag instead of an `h5` tag."